src/elm/Claim.elm

```diff
@@ -389,6 +387,242 @@ viewClaimCard { shared, accountName } profileSummary claim =
+                    [ text claimStatusPhrase
+                    ]
+                , div [ class textColor ] [ text claimStatus ]
+                ]
+
+        rewardInfo =
+            let
+                rewardTxtStyle =
+                    "my-2 font-bold text-lg"
+            in
+            div
+                [ class "text-center flex justify-center bg-gray-100 rounded-md p-4 space-x-16" ]
+                [ div
+                    []
+                    [ p [ class rewardTxtStyle ] [ text (String.fromFloat claim.action.reward) ]
+                    , p [ class greenTextTitleStyle ] [ text (t "community.actions.reward") ]
+                    ]
+                , div []
+                    [ p [ class rewardTxtStyle ] [ text (String.fromFloat claim.action.verifierReward) ]
+                    , p [ class greenTextTitleStyle ] [ text (t "claim.analyst_reward") ]
+                    ]
+                ]
+
+        approvedBy =
+            div
+                [ class claimVerifiersList ]
+                [ p [ class greenTextTitleStyle ] [ text (t "claim.approved_by") ]
+                , div [ class "overflow-x-auto" ]
+                    [ if List.isEmpty claim.checks then
+                        div [ class "flex mb-10" ]
+                            [ div [ class "pt-2" ] [ Profile.viewEmpty loggedIn.shared ]
+                            ]
+
+                      else
+                        div [ class "flex -mx-2 pt-2" ]
+                            (List.map
+                                (\c ->
+                                    if c.isApproved then
+                                        div [ class "px-2" ]
+                                            [ Profile.view loggedIn.shared loggedIn.accountName c.validator
+                                            ]
+
+                                    else
+                                        text ""
+                                )
+                                claim.checks
+                            )
+                    ]
+                ]
+
+        disapprovedBy =
+            div
+                [ class claimVerifiersList ]
+                [ p [ class greenTextTitleStyle ] [ text (t "claim.disapproved_by") ]
+                , div [ class "overflow-x-auto" ]
+                    [ if List.filter (\check -> check.isApproved == False) claim.checks |> List.isEmpty then
+                        div [ class "flex mb-10 " ]
+                            [ div [ class "pt-2" ] [ Profile.viewEmpty loggedIn.shared ] ]
+
+                      else
+                        div [ class "flex -mx-2 pt-2" ]
+                            (List.map
+                                (\c ->
+                                    if not c.isApproved then
+                                        div [ class "px-2" ]
+                                            [ Profile.view loggedIn.shared loggedIn.accountName c.validator
```

Comment:
After #537, Profile.view and Profile.viewLarge have been replaced by the functions in Profile.Summary - you need to keep track of a Profile.Summary.Model and have it update on your update function (this is that nested TEA thing you were doing on Neutro!)

This one is trickier because it's actually multiple of them. [Here](https://github.com/cambiatus/frontend/blob/master/src/elm/Page/Dashboard/Analysis.elm#L345-L347) is an example with multiple of them being handled